src/utils/get_last_visit_time.js

```diff
@@ -1,30 +1,34 @@
-export default function getLastVisitTime(timestamp) {
+export default function getLastVisitTime(timestamp, userLocale) {
```

Comment:
hmm, maybe such libs as date-fns (https://date-fns.org/docs/Getting-Started) or luxon (https://moment.github.io/luxon/#/?id=luxon) help you to simplify time formatting